web/web-portlet/src/bundle/iframe.properties

```diff
@@ -38,6 +38,7 @@ gen.custheight = Custom height
 gen.pixels = pixels
 gen.pixelshid = (in pixels)
 gen.info.pop = Open in new window?
+gen.popup.info=Note: Applies only to secure (https://) URLs. A non-secure (http://) URL will always open in a new window.
 ### (moot) gen.info.yes = yes
```

Comment:
I would go even further: Applies only to secure URLs that allow framing. Most major websites do not allow framing.